CHANGELOG.md

```diff
@@ -2,6 +2,32 @@
 
 ## Version 7
 
+### v7.5.0
+
+- Feature #503: configurable CORS headers.
+  - The configuration options `cors` now accepts a function that returns custom headers.
+  - The function may be asynchronous.
+  - Setting `cors: true` implies the default headers.
+
+```typescript
+import { createConfig } from "express-zod-api";
+
+const config = createConfig({
+  // ...
+  cors: ({ defaultHeaders, request, endpoint, logger }) => ({
```

Comment:
great, it permit to rewrite any headers, can rename from `cors` to `rewriteHeaders`